src/libraries/icubmod/embObjMotionControl/embObjMotionControl.h

```diff
@@ -256,7 +261,7 @@ class yarp::dev::embObjMotionControl:   public DeviceDriver,
     double  *_ref_accs;         // for velocity control, in position min jerk eq is used.
     double  *_encodersStamp;                    /** keep information about acquisition time for encoders read */
     bool  *checking_motiondone;                 /* flag telling if I'm already waiting for motion done */
-    #define MAX_POSITION_MOVE_INTERVAL 0.080
+    #define MAX_POSITION_MOVE_INTERVAL 0.0001
```

Comment:
What is this for?

Reply:
I changed it to avoid printing the warning message:
"Performance warning: You are using positionMove commands at high rate (<"<<MAX_POSITION_MOVE_INTERVAL*1000.0 <<" ms)"

I will revert it to its original value.

Reply:
Yes thanks, revert it.